app/models/concerns/indexable.rb

```diff
@@ -1,14 +1,14 @@
 module Indexable
   extend ActiveSupport::Concern
 
-  INDEX_NAME = [ENV["ALGOLIA_INDEX_PREFIX"], DOMAIN, Vacancy].compact.join("-").freeze
+  INDEX_NAME = [Rails.configuration.algolia_index_prefix, DOMAIN, Vacancy].compact.join("-").freeze
 
   included do
     include AlgoliaSearch
 
     scope :unindexed, (-> { live.where(initially_indexed: false) })
 
-    algoliasearch index_name: INDEX_NAME, auto_index: true, auto_remove: true, if: :listed? do
+    algoliasearch index_name: INDEX_NAME, auto_index: !Rails.env.test?, auto_remove: !Rails.env.test?, if: :listed? do
```

Comment:
I wouldn't want to make the test behaviour different from the real behaviour without good reason. What's the reason?

Reply:
If we don't do this then we'll be making requests to Algolia automatically every time we create a vacancy and they won't have been recorded in VCR cassettes

Reply:
Which means lots and lots and lots of errors